packages/lab/src/Wizard/stories/Wizard.stories.js

```diff
@@ -0,0 +1,531 @@
+          lobortis metus blandit et. Suspendisse mollis, enim in maximus lobortis, sem elit pharetra
+          nisl, quis elementum est libero sit amet turpis. Suspendisse felis elit, venenatis in
+          semper in, imperdiet id est. Etiam est massa, dignissim vel ipsum sit amet, ultricies
+          mattis leo. Nullam lacinia id est id iaculis. Vivamus ligula tortor, sagittis eu ipsum
+          eget, tempor consectetur odio. Vestibulum sed orci dictum, finibus eros vel, consequat
+          nulla. Pellentesque ac leo vitae enim gravida laoreet vestibulum ac neque. Sed in mattis
+          nunc, vitae eleifend turpis. Duis et cursus lorem, et luctus nibh. Duis nisi velit, cursus
+          id bibendum at, gravida in nulla. Nunc bibendum ligula quis leo condimentum sollicitudin.
+          Donec at tristique mauris. Cras in ligula placerat, imperdiet ipsum eget, ullamcorper ex.
+          Nulla bibendum urna nec nunc blandit ornare. Quisque dolor erat, ultrices quis lacus id,
+          bibendum pretium urna. Phasellus a odio vitae dolor convallis egestas. Interdum et
+          malesuada fames ac ante ipsum primis in faucibus. In suscipit urna ac elit dictum aliquet.
+          Etiam vitae sodales magna, eget sodales leo. Nam molestie vulputate nisl sit amet
+          eleifend. Aliquam erat volutpat. Phasellus sit amet ultricies tortor, et malesuada lacus.
+          Donec euismod ex et ipsum iaculis, eget consectetur purus vestibulum. Aliquam eu accumsan
+          purus, pretium volutpat tortor. Nam gravida vitae ipsum et porta. Sed lectus arcu, viverra
+          in tincidunt et, tempus quis libero. Nullam vitae fringilla turpis. Fusce pretium
+          porttitor aliquet. In sapien orci, scelerisque ut malesuada eget, suscipit et dui. Aliquam
+          erat volutpat. Phasellus sed commodo felis. Aenean leo metus, ultricies sit amet sapien
+          sed, hendrerit vehicula lectus. Mauris feugiat, est ut tempor scelerisque, dolor diam
+          condimentum sem, at pellentesque arcu erat mattis lectus. Nulla maximus lorem sit amet
+          gravida malesuada.
+        </div>
+        <div name="last">Ultimo</div>
+      </HvWizard>
+    </>
+  );
+};
+
+export const Sample2 = () => {
+  const [open, setOpen] = React.useState(false);
+  const [tab, setTab] = React.useState(0);
+  const [context, setContext] = React.useState({});
+
+  const contextValue = React.useMemo(
+    () => ({
+      context,
+      setContext,
+    }),
+    [context, setContext]
+  );
+
+  const handleClose = React.useCallback((evt, reason) => {
+    if (reason !== "backdropClick") {
+      setTab(0);
+      setOpen(false);
+    }
+  }, []);
+
+  return (
+    <>
+      <HvButton onClick={() => setOpen(true)}>Show Wizard</HvButton>
+      <HvWizardContext.Provider value={contextValue}>
+        <HvWizardContainer open={open} handleClose={handleClose}>
+          <HvWizardTitle title="Super component" tab={tab} />
```

Comment:
You have to declare prop changeTab prop on HvWizardTitle to remove a console warning